cluster-operator/src/main/java/io/strimzi/operator/cluster/FeatureGates.java

```diff
@@ -17,9 +17,12 @@ public class FeatureGates {
     /* test */ static final FeatureGates NONE = new FeatureGates("");
 
     private static final String USE_KRAFT = "UseKRaft";
+    private static final String CONTINUE_RECONCILIATION_ON_MANUAL_ROLLING_UPDATE_FAILURE = "ContinueReconciliationOnManualRollingUpdateFailure";
 
     // When adding new feature gates, do not forget to add them to allFeatureGates() and toString() methods
     private final FeatureGate useKRaft = new FeatureGate(USE_KRAFT, true);
+    private final FeatureGate continueReconciliationOnManualRollingUpdateFailure =
```

Comment:
I wonder if we should try to shorten it up a bit. We should keep the feature gate name as is (`ContinueReconciliationOnManualRollingUpdateFailure`). But what about using `CONTINUE_ON_MANUAL_RU_FAILURE` and `continueOnManualRUFailure` for the variable and the related method? It will make it a bit easier to read the code and I think it would be still readable.